modules/standard/DateTime.chpl

```diff
@@ -434,6 +434,13 @@ module DateTime {
     return str;
   }
 
+  /* Read or write a date value from channel `f` */
+  proc date.readWriteThis(f) {
+    var dash = new ioLiteral("-");
+    f <~> new ioLiteral("{") <~> chpl_year <~> dash <~> chpl_month <~> dash
+      <~> chpl_day <~> new ioLiteral("}");
```

Comment:
Is there a specific reason for the { } around the date? I'd think 2018-03-12 is clear enough as a date already. Are you following some Python precedent, e.g.?

Reply:
I was following the default behavior of Chapel classes, which doesn't make a whole lot of sense given that these are records.  I'll remove the curly braces.